packages/client/test/cli/cli.spec.ts

```diff
@@ -3,19 +3,46 @@ import { Client } from 'jayson/promise'
 import { assert, describe, it } from 'vitest'
 
 describe('[CLI]', () => {
-  it('should start HTTP RPC and return valid responses', async () => {
+  // chain network tests
+  it('should successfully start client with a custom network and network id', async () => {
     const file = require.resolve('../../dist/bin/cli.js')
-    const cliArgs = ['--rpc', '--dev=poa']
+    const cliArgs = ['--network=sepolia', '--networkId=11155111', '--dev=poa']
+    const child = spawn(process.execPath, [file, ...cliArgs])
+    return new Promise((resolve) => {
+      child.stdout.on('data', async (data) => {
+        const message: string = data.toString()
+        if (message.includes('Client started successfully')) {
+          assert.ok('Client started successfully', 'read from HTTP RPC')
```

Comment:
These test success messages seem a bit imprecise.  Can you adjust the second message so it reflects the what the test is actually doing?  As an example, in this one, the test isn't that that the RPC started but that the client started successfully and that it's using a custom network setup.

Reply:
Also, this test isn't actually working correctly.  If you combine the network parameters with `--dev=poa`, the devnet parameters take precedent.  You can log the output of the messages to the console by adding something like
```js
      child.stdout.on('data', async (data) => {
        console.log(data.toString())
```

Then, you can adjust the tests accordingly to check for valid log messages that correspond to the test parameters you're working with.

In this test, we should be validating that the below log message has:
`[07-27|06:13:01] INFO Initializing Ethereumjs client version=v0.8.0 network=sepolia chainId=11155111`

but right now it is showing:
`[07-27|06:13:01] INFO Initializing Ethereumjs client version=v0.8.0 network=devnet chainId=123456`

Reply:
fixed